### ISSUE TITLE
Creating a form to enable non-programmers to add websites

### ISSUE THREAD

**Issue by ItayXD** [2factorauth/twofactorauth#3163]
I want to know if you'll be interested in adding a form which will enable non-programmers to add websites to the lists without dealing with GitHub yaml or pull request?

**Comment by mxxcon** [comment]
There recently was another discussion on this issue.
How do you foresee this working?

**Comment by kenman345** [comment]
Acceptbitcoincash repo that forked from this one, uses a google form to make new issues on our repository to inform us of sites that need to be added to our listings and the yaml formatted nearly perfectly based on the answers to the questions in the form.

here is an example: acceptbitcoincash/acceptbitcoincash#912
Here is the web form: [Request form](https://acceptbitcoin.cash/add)

I've offered something like this in the past, but was not given much to go off of. 

To facilitate the entry of those requests, I also wrote a few Rake tasks that allow someone that helps maintain the site to open a command line, run the task, input the issue number and save the image to the proper place. Its quite nice and saves some time for our process. It uses the `website_schema.yml` file in the root of the repository as well to make sure it validates the request structure.

Only problem with the rake task is when it outputs the yaml it added into, it changes the spacing like this file did: https://github.com/acceptbitcoincash/acceptbitcoincash/commit/d354e099d251673527d3f64d6eda6a111be2fb54#diff-868c9235147be5932dd11b6f23ee515b
